exporter/clients.go

```diff
@@ -71,11 +72,12 @@ func parseClientListString(clientInfo string) (*ClientInfo, bool) {
 			connectedClient.Cmd = vPart[1]
 		case "addr":
 			hostPortString := strings.Split(vPart[1], ":")
-			if len(hostPortString) != 2 {
+			if len(hostPortString) < 2 {
+				log.Debug("Invalid value for 'addr' found in client info")
 				return nil, false
 			}
-			connectedClient.Host = hostPortString[0]
-			connectedClient.Port = hostPortString[1]
+			connectedClient.Host = strings.Join(hostPortString[:len(hostPortString)-1], ":")
```

Comment:
How's that working with IPv4? In that case we just have one item in the "Join()" and no ":" will be added?

Reply:
Hi @oliver006 - thanks for the quick review!

Yes, that's correct. When there's 1 item in `Join` the join character is not added.